saleor/graphql/file_upload/views.py

```diff
@@ -8,6 +9,18 @@
 
 class FileUploadGraphQLView(GraphQLView):
 
+    def dispatch(self, request, *args, **kwargs):
+        # Handle options method the GraphQlView restricts it.
+        if request.method.lower() == 'options':
+            response =  self.options(request, *args, **kwargs)
+        else:
+            response = super().dispatch(request, *args, **kwargs)
+        # Add access control headers 
+        response['Access-Control-Allow-Origin'] = ','.join(settings.ALLOWED_HOSTS)
```

Comment:
Do we need to limit this to a specific list of hosts? Most APIs just return `Access-Control-Allow-Origin: *`.

Reply:
I think it would still be better to allow the user to provide their own value if they want to only allow a few hosts. And default the value to `*`.